packages/ag-charts-website/.env.build.staging

```
@@ -5,3 +5,4 @@ PUBLIC_ALGOLIA_APP_ID=O1K1ESGB5K
 PUBLIC_ALGOLIA_SEARCH_KEY=01142fe24ea5d2e36f4eb66b0b2ff872
 PUBLIC_ALGOLIA_INDEX_PREFIX=ag-charts-dev
 PUBLIC_PACKAGE_VERSION=10.3.3-beta.20241202
+LIVE_SITEMAP_URL=https://ag-grid.com/charts/sitemap-0.xml
```

This can probably point to charts staging

```suggestion
LIVE_SITEMAP_URL=https://charts-staging.ag-grid.com/sitemap-0.xml
```